src/reducers/run-status.js

```diff
@@ -0,0 +1,8 @@
+import { statusMockData } from '../utils/status-mock-data';
+import { datasetError } from '../utils/run-status-mock-data/dataset-error';
+import { workingAllGreen } from '../utils/run-status-mock-data/working-all-green';
+import { funNodeError } from '../utils/run-status-mock-data/fun-node-error';
+
+export default function runStatus(state = funNodeError || {}, action) {
```

Comment:
this will be updated according this new runStatus reducer https://github.com/kedro-org/kedro-viz/blob/64da498646a872475622d45e569b59d5eac4b99e/src/reducers/run-status.js